app/Http/Requests/Settings/ChangeLoginRequest.php

```diff
@@ -52,7 +49,6 @@ public function authorize(): bool
 	public function rules(): array
 	{
 		$rules = parent::rules();
-		$rules[self::OLD_USERNAME_ATTRIBUTE] = ['sometimes', new UsernameRule()];
```

Comment:
I see that you have removed the old username from the request and from the action which handles the password setting.

Have you also changed the frontend accordingly? I had commented in the action that I regarded the old user name as suspicious, but I did not dare to change it. What possible consequences has the removal of the old user name?

Reply:
I dared. :)